back/spec/models/organization_user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe OrganizationUser, type: :model do
+  let(:organization) { FactoryBot.create(:organization, :with_user) }
+
+  describe '必須項目' do
+    it 'organization_id が空の場合エラーになること' do
+      organization_user = OrganizationUser.new(organization: nil, user: organization.users.first)
+      expect(organization_user.valid?).to be false
+      expect(organization_user.errors[:organization]).to include("must exist")
+    end
+    it 'user_id が空の場合エラーになること' do
+      organization_user = OrganizationUser.new(organization:, user: nil)
+      expect(organization_user.valid?).to be false
+      expect(organization_user.errors[:user]).to include("must exist")
+    end
+  end
+
+  describe '重複' do
+    it 'organizationとuser の組み合わせが重複する場合エラーになること' do
+      organization_user = OrganizationUser.create(organization:, user: organization.users.first)
+      another_organization_user = OrganizationUser.new(organization:, user: organization_user.user)
```

Comment:
`OrganizationUser.new(organization:`でOrganizationUserが作成されるはずなので、冗長かなと思います
もしくは`let(:organization) { FactoryBot.create(:organization, :with_user) }`を`let!`にして`describe '重複'`のブロック内に移動してもいいかもです
```suggestion
      another_organization_user = OrganizationUser.new(organization:, user: organization.users[0])
```